endpoints.md

```diff
@@ -0,0 +1,108 @@
+## Claim Post
+
+Url: /claim
+
+Method: Any
+
+Accepted JSON fields:
+
+| Field Name      | Required | Content                      |
+|-----------------|----------|------------------------------|
+| api_key         | Yes      | String; the api key          |
+| name            | Yes      | String; the user's name      |
+| is_admin        | No       | Boolean; is an admin or not  |
+| debug           | No       | Int; debug to fixed output   |
+
+## Done Post
+
+Url: /done
+
+Method: Any
+
+Accepted JSON fields:
+
+| Field Name      | Required | Content                      |
+|-----------------|----------|------------------------------|
+| api_key         | Yes      | String; the api key          |
+| post_id         | Yes      | String; the post id          |
+| debug           | No       | Int; debug to fixed output   |
+
+## Unclaim Post
+
+Url: /unclaim
+
+Method: Any
+
+Accepted JSON fields:
+
+| Field Name      | Required | Content                      |
+|-----------------|----------|------------------------------|
+| api_key         | Yes      | String; the api key          |
+| post_id         | Yes      | String; the post id          |
+| debug           | No       | Int; debug to fixed output   |
+
+## Create Keys
+
+Url: /keys/create
+
+Method: Post
+
+Accepted JSON fields:
+
+| Field Name    | Required | Content                      |
+|---------------|----------|------------------------------|
+| api_key       | Yes      | String; the api key          |
```

Comment:
This is an admin endpoint; it requires a key with the admin flag. (Or alternatively, just "This is an admin-only endpoint")